util/filter.lua

```diff
@@ -21,6 +21,24 @@ function default_filter(str)
 end
 
 M.filters = {
+    ['not'] = {
+        input_type = '',
+    },
+
+    ['and'] = {
+        input_type = 'number',
+        validator = function(arity)
+            return arity and arity > 0
```

Comment:
What about the default (infinite) arity?

Reply:
If I understood correctly and under the default (infinity) arity, you mean the case when the arity is not specified, then everything works correctly with the implementation of `arity and arity > 0`.
The only thing I don't like is that it is impossible to indicate as an arity value "1" (which is equivalent when the arity is not specified). I do not know how to solve this problem, but I think this is not a significant problem.